docker/k8s/fabric/blockchain-connector.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: avalon-blockchain-connector
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: avalon-blockchain-connector
+  template:
+    metadata:
+      name: avalon-blockchain-connector
+      labels:
+        app: avalon-blockchain-connector
+    spec:
+      containers:
+      - image: avalon-blockchain-connector-dev:latest
+        name: avalon-blockchain-connector
+        imagePullPolicy: Never
+        command: ["/bin/bash"]
+        args: ["-c", "avalon_blockchain_connector -b fabric -u http://avalon-listener:1947"]
+        volumeMounts:
+        - name: blockmark
+          mountPath: /project/avalon/blockchain_connector/blockmark
+        - name: keyfiles
+          mountPath: /keyfiles
+        - name: network
+          mountPath: /project/avalon/sdk/avalon_sdk/connector/blockchains/fabric/network.json
+      volumes:
+      - name: blockmark
+        hostPath:
+          path: /home/ubuntu/mywork/vars/blockmark
```

Comment:
Will this solution work, if block chain connector pods gets deployed in different nodes ? Is this path configured by `start_fabric.sh` ? If so, please add a section in documentation with assumptions and current limitation

Reply:
This is the limitation with both shell and blockchain connector containers. So is with k8s pods.

Reply:
Though start_fabric.sh allows to install minifab at a place of our choice. I am going to work on that in my next PR to improve on it.

Reply:
Initial PR can assume single node setup and we can document this limitation. But we need to discuss how to address this limitation in Proxy model since we need to support k8s pods with multiple nodes. Direct model k8s pods works with multiple nodes.

Reply:
I agree. This PR doesn't target that. I will be on this task to remove this limitation as a next step.

Reply:
All we need to share the crypto materials generated from minifab needs to be shared to avalon-shell and blockchain connector then, we are good run in multiple machines.